usr/src/lib/libzpool/Makefile.com

```diff
@@ -61,6 +61,7 @@ CLEANFILES += $(EXTPICS)
 
 $(LINTLIB) := SRCS=	$(SRCDIR)/$(LINTSRC)
 $(LINTLIB): ../common/zfs.h
```

Comment:
i have added it as separate target because if you want remove LINTLIB target at all - build will be failed - and you need update this Makefile again.
also, right now, you can run only $(LINTLIB) target if needed and build will be fine.
i just try to fix build without lint - what i can test.
also - 2 dependency in LIBS - like for consistency with others - copy/past - and it is OK.
problem only with header what is using for dtrace build